components/bulletins/BulletinCard.vue

```diff
@@ -50,55 +50,56 @@ export default {
                 bulletinCard: true,
             }
         },
+        showButton() {
+            return false
+        },
     },
 }
 </script>
 
 <style lang="postcss" scoped>
 .bulletinCard {
-    @apply relative flex flex-col items-center font-serif rounded-3xl border-4;
-    width: 47%;
-    @media (min-width: 415px) {
-        width: 214px;
-    }
-    border-color: #4b4b4b;
-    color: #4b4b4b;
+    @apply relative flex flex-col justify-center items-center rounded-xl;
+    min-width: 236px;
+    min-height: 310px;
+    color: #a9a6d9;
+    background: #1f1c3b;
 }
 
 .bulletinCard:hover {
-    border-color: #413f5a;
+    border-color: #746bb8;
+    background: #352d66;
+    color: #f0ebf5;
```

Comment:
**issue (blocking):** there are some other changes like the different blur, icon color, font color, and box-shadow when hovering. Please check this [spec](https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=1970%3A58579).

Reply:
Fixed except the icon color